README.md

```diff
@@ -1,2 +1,12 @@
-# website
+# CHAOSS Website - Content Requests and Edits
+Please use the Issues Tab to submit content requests, edits, and enhancements. 
```

Comment:
How about promoting pull-requests --- enables other's to directly contribute text and streamlines the process of getting it into the website. Reduce the work required by the maintainer --- you.

Reply:
Are you suggesting putting the web site code on GitHub or just building a file structure on the repo that would allow for text editing for select areas?

Reply:
I suggest the latter. Put the editable text on GitHub. I assume that pull-requests reduce the burden on the maintainers.

Reply:
Similar to we have a copy of the governance document on GitHub.

Reply:
Sounds Good

Reply:
@GeorgLink do you mean using GitHub Pages as website for chaoss.community, or something else?

Reply:
@jgbarah,  If I am understanding it correctly, he is advocating for a md file where contributors can provide and edit text that we can then copy into the Wordpress site. For example we can have a news.md page or folder where contributors can write a blog post to appear on the site.  @GeorgLink, is that correct?

Reply:
@jgbarah I propose something else, not GitHub Pages.

As @klumb says: I propose to mirror the text here because editing it with a pull-request is more direct than describing changes in an issue that then the maintainer has to figure out how to incorporate it into existing text.

I don't know if it makes sense to mirror the text of all pages here, maybe we can start with the ones that we want to work on. 

I also don't know whether Markdown, Text, HTML, or another format is the best to store in this repository because I don't know how easy or difficult each format is for updating the Wordpress installation.

Reply:
I see. Maybe it is going to be a bit of a problem to make all modifications here, and then "transfer" them to WordPress, with (I think) a different markup. But we can try, or use it only for new pages. I also prefer the "pull request way" of suggesting modifications.

Reply:
I suggest we try it for the news/blog posts and add other content pages as needed.  I think we can still use the issues area as described in the current ReadMe for discussions and requests.